CybeReason/malop/tests/malop_machine.json

```
@@ -0,0 +1,58 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "cybereason-malop",
+        "dialect_uuid": "0de050fb-3f56-4c7a-a9b6-76bf5298a617"
+      }
+    },
+    "message": "CEF:0|Cybereason|Cybereason|1.0|3|Malop Machine Added|5|destinationDnsDomain=desktop-aas6kq7 dst=10.0.2.15 destinationTranslatedAddress=117.99.232.147 CybereasonCEFgeneratorBatchId1=2ac124fd-def2-4073-b408-d3b3f0e764b0 cs1=11.-6654920844431693523 flexString2=True dhost=desktop-aas6kq7 CybereasonCEFgeneratorOSandVersion1=Windows_10 CybereasonCEFgeneratorMachineGuid1=-592942600.1198775089551518743 cfp3=1 rt=1625748509151 cfp2=1 cs1Label=MalopId flexString2Label=isMalicious cfp2Label=isOnline cfp3Label=isOriginalMachine request=\"C:\\\\Users\\\\chand\\\\Downloads\\\\BT_21.40.5_32_Win7.pdf.exe\" deviceProcessName=explorer.exe CybereasonCEFgeneratorChildProcess1=None"
+  },
+  "expected": {
+    "@timestamp": "2021-07-08T12:48:29.151000Z",
+    "message": "CEF:0|Cybereason|Cybereason|1.0|3|Malop Machine Added|5|destinationDnsDomain=desktop-aas6kq7 dst=10.0.2.15 destinationTranslatedAddress=117.99.232.147 CybereasonCEFgeneratorBatchId1=2ac124fd-def2-4073-b408-d3b3f0e764b0 cs1=11.-6654920844431693523 flexString2=True dhost=desktop-aas6kq7 CybereasonCEFgeneratorOSandVersion1=Windows_10 CybereasonCEFgeneratorMachineGuid1=-592942600.1198775089551518743 cfp3=1 rt=1625748509151 cfp2=1 cs1Label=MalopId flexString2Label=isMalicious cfp2Label=isOnline cfp3Label=isOriginalMachine request=\"C:\\\\Users\\\\chand\\\\Downloads\\\\BT_21.40.5_32_Win7.pdf.exe\" deviceProcessName=explorer.exe CybereasonCEFgeneratorChildProcess1=None",
+    "observer": {
+        "vendor": "Cybereason",
+        "product": "Cybereason",
+        "version": "1.0"
+    },
+    "event": {
+        "severity": 5,
+        "code": "3",
+        "id": "2ac124fd-def2-4073-b408-d3b3f0e764b0",
+        "kind": "event",
+        "type": "info",
+        "category": ["intrusion_detection"],
+        "action": "Malop Machine Added"
+    },
+    "destination": {
+        "ip": "10.0.2.15",
+        "nat": {"ip": "117.99.232.147"}
+    },
+    "host": {
+        "hostname": "desktop-aas6kq7",
+        "ip": ["10.0.2.15", "117.99.232.147"],
+        "id": "-592942600.1198775089551518743",
+        "os": {
+            "full": "Windows 10"
+        }
+    },
+    "process": {
+        "command_line": "\"C:\\\\Users\\\\chand\\\\Downloads\\\\BT_21.40.5_32_Win7.pdf.exe\"",
```

Shouldn't it be something like:

```suggestion
        "command_line": "C:\\Users\\chand\\Downloads\\BT_21.40.5_32_Win7.pdf.exe",
```